packages/kit/src/core/sync/write_non_ambient.js

```diff
@@ -26,7 +26,7 @@ declare module "svelte/elements" {
 			| null;
 		'data-sveltekit-preload-data'?: true | '' | 'hover' | 'tap' | 'off' | undefined | null;
 		'data-sveltekit-reload'?: true | '' | 'off' | undefined | null;
-		'data-sveltekit-replacestate'?: true | '' | 'off' | undefined | null;
+		'data-sveltekit-replace_state'?: true | '' | 'off' | undefined | null;
```

Comment:
Not sure if this change is intended, but should it be a hypen?

```suggestion
		'data-sveltekit-replace-state'?: true | '' | 'off' | undefined | null;
```

Reply:
oops! no, this was a copy-paste fail